### DISCUSSION TITLE
<Table.Th> should respect "align" property

### DISCUSSION THREAD

**Discussion by baba43** [mantinedev/mantine#6550] [Feature requests]
Currently this code works as intended:

`<Table.Td align={"right"}>Views</Table.Td>`

But this code does not:

`<Table.Th align={"right"}>Views</Table.Th>`

Not sure if this should be the case at all.. just want to note it. 

I _fixed_ it by providing this CSS to make it consistant:

```
.mantine-Table-td,
.mantine-Table-th {
    &[align="right"] {
        text-align: right;
    }
}
```

**Comment by rtivital** [comment]
Why you cannot use `ta="right"` instead?

**Reply by baba43** [reply]
Oh, I can. Great. Thank you 👍
